packages/react-components/src/components/VideoTile.tsx

```diff
@@ -86,6 +93,11 @@ export interface VideoTileProps {
    * Whether the video is muted or not.
    */
   isMuted?: boolean;
+  /* @conditional-compile-remove(pinned-participants) */
+  /**
+   * Display custom menu items in the VideoTile's contextual menu.
+   */
+  menuItems?: Array<VideoTileMenuItem>;
```

Comment:
We can use the type VideoTileMenuItems you declared below. `VideoTileMenuItems`

Reply:
done